kubernetes-manifests/environment-config.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: environment-config
+data:
+  VERSION: "v0.1.0"
+  LOCAL_ROUTING_NUM: "123456789"
+  PUB_KEY_PATH: "/root/.ssh/publickey"
+  PRIV_KEY_PATH: "/root/.ssh/privatekey"
+  TRANSACTIONS_API_ADDR: "ledgerwriter:8080"
+  BALANCES_API_ADDR: "balancereader:8080"
+  HISTORY_API_ADDR: "transactionhistory:8080"
+  CONTACTS_API_ADDR: "contacts:8080"
+  USERSERVICE_API_ADDR: "userservice:8080"
```

Comment:
Some services like accounts-db use LOCAL_ROUTING_NUM, but not the service urls. Maybe we should break down this map further

Reply:
Alrighty, let's do it.